### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ievans**
Add a link to semgrep-rules under the patterns field

### CODE REVIEW

**Review by brendongo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ Example patterns:
 | `$ELEMENT.innerHTML` | ``el.innerHTML = "<img src='x' onerror='alert(`XSS`)'>";`` |
 | `$TOKEN.SignedString([]byte("..."))` | `ss, err := token.SignedString([]byte("HARDCODED KEY"))` |
 
+→ [see more example patterns in the semgrep-rules repository](https://github.com/returntocorp/semgrep-rules)
```
Comment (pabloest) [inline README.md:31]: I think this works fine but how about sending people to https://semgrep.live/r/ ? Just seems easier to browse than the GitHub file navigation IMO.

Comment (ievans) [reply README.md:31]: @DrewDennison would have to comment--AFAIK that's not ready to be advertised yet?

Comment (DrewDennison) [reply README.md:31]: @pabloest I think https://semgrep.live/r/ is still too functional without enough context and explanation for direct linking. I would like to hustle this weekend on making it so it's ready and I know Luke has been asking for a new design on that page. If you want to spec it out with me, that would be great

Comment (pabloest) [reply README.md:31]: Happy to help, @DrewDennison. We can chat off-thread.

**Pull request merged**